tidal.py

```diff
@@ -17,8 +17,75 @@ def env_constructor(loader, node):
 
 yaml.SafeLoader.add_constructor("!env", env_constructor)
 
-if __name__ == '__main__':
-    with open("config.yml", 'r') as f:
+
+async def build_track_entry(builder, el):
+    title = (
+        el["title"]
+        if "version" not in el or not el["version"]
+        else f"{el['title']} ({el['version']})"
+    )
+
+    artists = ", ".join(map(lambda artist: artist["name"], el["artists"]))
+    artists_label = f"Artists: {artists}" if len(artists) > 1 else f"Artist: {artists}"
+
+    album_label = f"Album: {el['album'['title']]}"
+
+    explicit = (
+        el["properties"]["content"] == "explicit"
+        if hasattr(el, "properties") and hasattr(el["properties"], "content")
```

Comment:
for sure not hasattr, you can check for `properties in el` but it's not needed, you can use `explicit = el.get("properties", {}).get("content", "") == "explicit"`

Reply:
Will do, I was looking for something like this, I'm not really that into python 😁